### PR TITLE
October Edition

### PR DESCRIPTION
**Pull request by equivalentideas**
See [most up to date version of text](https://github.com/openaustralia/newsletter/blob/october-edition/editions/2015-10.md).

### CODE REVIEW

File: editions/2015-10.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+##  News and Notces
```
Comment (henare) [inline editions/2015-10.md]: Typo.

File: editions/2015-10.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+### [Web scraping in Ruby tutorial](https://www.openaustraliafoundation.org.au/tag/ruby-web-scraping-tutorial-on-morph-io/)
+
+Walk through creating a web scraper in Ruby and running it on morph.io
+in this [blog post series](mak://www.openaustraliafoundation.org.au/tag/ruby-web-scraping-tutorial-on-morph-io/).
+This is a detailed tutorial in which you
+actually create, publish and run your own working scraper.
+
+Give it a try and let us know how you go.
+Through recourses like this, and our workshops,
```
Comment (henare) [inline editions/2015-10.md]: Typo.

**Comment by equivalentideas** [comment]
> I think we should either cut this or add other links to other articles to this item.

I'm going to cut it :scissors:

**Comment by equivalentideas** [comment]
Added intro and checked links. I'm going to merge and :shipit: .

**Pull request merged**